cocos/core/pipeline/instanced-buffer.ts

```diff
@@ -138,7 +138,8 @@ export class InstancedBuffer {
         vertexBuffers.push(vb);
         const iaInfo = new InputAssemblerInfo(attributes, vertexBuffers, indexBuffer);
         const ia = this._device.createInputAssembler(iaInfo);
-        this.instances.push({ count: 1, capacity: INITIAL_CAPACITY, vb, data, ia, stride, hShader, hDescriptorSet, lightingMap});
+        let wireframeIa;
```

Comment:
What's the value of `wireframeIa `?

Reply:
wireframeIa is the inputassembler object of wireframe function, it will be assigned in wireframequeue